churner/churner.go

```diff
@@ -47,7 +47,7 @@ type churner struct {
 	mu          sync.Mutex
 	Indexer     thegraph.IndexedChainState
 	Transactor  core.Transactor
-	QuorumCount uint16
+	QuorumCount uint8
```

Comment:
The max of uint8 is 255, so it can have 256 quorums, and uint8 cannot represent 256.

Reply:
The onchain quorumCount was changed to a uint8 (https://github.com/Layr-Labs/eigenlayer-middleware/blob/98f884454d9e9de1e344bb6fba9a2cd3915e5b57/src/RegistryCoordinator.sol#L71), so it looks like we would only support up to 255 quorums.

Reply:
ok, so the range of quorum ID is [0, 254] (not [0, 255]).